GO_stored_queries/GO_get_hierarchicalAncestors.yaml

```diff
@@ -0,0 +1,17 @@
+# Get all hierarchical ancestors 
+# (all parents’s parents) resources for this term. Hierarchical 
+# ancestors include is-a and other related parents, such as 
+# part-of/develops-from, that imply a hierarchical relationship
+
+name: GO_get_hierarchicalAncestors
+params:
+  type: object
+  required: [key]
+  properties:
+    key:
+      type: string
+      title: Document key
+      description: Get all hierarchical ancestors for this term
+query: |
+  WITH GO_test_term FOR v, e IN 1..1000000 OUTBOUND @key GO_test_edges_isa, GO_test_edges_relationship
+      RETURN {term: v, edge: e}
```

Comment:
Should we filter on relationship type? I'm reading this comment: `Hierarchical ancestors include is-a and other related parents, such as part-of/develops-from, that imply a hierarchical relationship` -- and I'm thinking not all relationships are hierarchical, so we need to filter those out.

We might need two separate queries that we return on a union: isa query, and a separate relationship query that filters based on `.relationship_type`.

Reply:
Oh right. The hierarchical relationships seem to be has_part and part_of. Is occurs_in hierarchical?

Relationships in GO: ends_during, happens_during, has_part, negatively_regulates, never_in_taxon, occurs_in, part_of, positively_regulates, regulates, starts_during.